tests/qchem/test_tapering.py

```diff
@@ -263,6 +290,16 @@ def test_cliford(generator, paulixops, result):
     u = clifford(generator, paulixops)
     assert u.compare(result)
 
+    # test arithmetic op compatibility:
```

Comment:
Can you add the missing `f` in the test name? Looks like I forgot that while implementing it.